compiler/rustc_hir_typeck/src/fn_ctxt/suggestions.rs

```diff
@@ -874,9 +874,9 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                 } else {
                     // Only point to return type if the expected type is the return type, as if they
                     // are not, the expectation must have been caused by something else.
-                    debug!("return type {:?}", hir_ty);
+                    debug!("return type (hir::Ty) {:?}", hir_ty);
                     let ty = self.lowerer().lower_ty(hir_ty);
-                    debug!("return type {:?}", ty);
+                    debug!("return type (ty) {:?}", ty);
                     debug!("expected type {:?}", expected);
```

Comment:
Btw, `tracing` has a nice shorthand for this but it doesn't matter (for future reference).

```rs
debug!(?hir_ty, "return type");
let ty = self.lowerer().lower_ty(hir_ty);
debug!(?ty, "return type {:?}");
debug!(ty = ?expected, "expected type");
```

which will print:

```
return type hir_ty=...
return type ty=...
expected type exp ty=...
```

Reply:
(I actually did know that shorthand, I just copied over what existed lol)

Reply:
... but now that you mentioned it, it's indeed nicer so I changed it ^^